### DISCUSSION TITLE
I am a new user, where can I find German-speaking support for my website?

### DISCUSSION THREAD

**Discussion by l4ndfr05ch** [datenstrom/community#945] [Ask a question] [answered]
Guten Abend,
vorab zur Info: ich bin absoluter Yellow-Neuling und kann auch nicht programmieren, möchte aber unbedingt ein Flat-CMS nutzen. 

Die Grundinstallation (also Blog) hat geklappt. Das Erstellen von neuen Seiten funktioniert auch bestens. Nun möchte ich aber das Layout etwas anpassen. Mir gefällt das Layout von der Datenstrom-Webseite sehr gut, also im Speziellen die oberste Zeile mit dem Suchfeld. 

Hier nun meine ganz konkreten Fragen:

1. Wie kann ich ein Suchfeld integrieren (ebenfalls auf der rechten Seite)?
2. Wie kann ich das Menü linksbündig ausrichten?
3. Wie kann man die Anzeige vom Webseiten-Titel (bei mir HOME) unterbinden? Ist ja identisch mit einem Klick auf STARTSEITE
4. Wie kann ich weitere Menüpunkte hinzufügen?
5. Gibt es eine Erweiterung für ein Pulldown-Menü?
6. Wie kann ich Kommentare unter meinen Blogartikeln zulassen?
7. Gibt es eine Möglichkeit Matomo o.ä. einzubinden?
8. Gibt es eine Möglichkeit (vllt am Ende der Seite) automatisiert eine Übersicht über alle verwendeten Tags zu zeigen?

Mein Blog: [landfrosch.com](https://www.landfrosch.com)

Viele Grüße,
Marko

**Comment by schulle4u** [comment, marked as answer]
Hallo Markus,

willkommen in der Yellow-Gemeinschaft! 👍 Zunächst der obligatorische Hinweis auf die [Richtlinien zum Zusammenarbeiten](https://datenstrom.se/de/yellow/help/contributing-guidelines), speziell der Hinweis für jede Frage eine neue Diskussion zu beginnen. Wir haben für dieses Mal den Diskussionstitel entsprechend angepasst. Die Verkehrssprache ist in der Regel englisch, dafür reicht auch notfalls ein [guter Übersetzer](https://deepl.com), wenn es mit eigenen Sprachkenntnissen hapert. 

Zu deinen Fragen kann ich allgemein das Lesen der Dokumentation empfehlen sowie die Orientierung an fertigen Erweiterungen. Speziell einige Themes bringen eigene Layouts mit und können dir einen Hinweis geben, [wie man Layouts anpasst](https://datenstrom.se/de/yellow/help/how-to-customise-a-layout). Der Rest ist eigentlich nur klassisches HTML und CSS. Bei solchen Anpassungen kommt man leider nicht darum herum sich einige allgemeine Kenntnisse anzueignen. 

Zum Seitenmenü: Yellow arbeitet auf Basis von Verzeichnissen und Dateien, woraus sich dann auch das Menü ergibt. Erstellt man eine neue Seite im Dateisystem oder über den Browser, dann erscheint auch ein neuer Eintrag im Seitenmenü. Jeder Unterordner im `content`-Verzeichnis enthält eine `page.md`, was bei einer statischen Webseite in etwa der `index.html` entspricht. Einzelne Ordner können auch mehrere Seiten enthalten, die tauchen dann aber nicht als Menüpunkt auf sondern müssen direkt verlinkt werden oder werden - wie im Fall der Blog-Erweiterung - über ein Layout eingebunden. 

Zu deinen Erweiterungsfragen schaue dir mal die [veröffentlichten Erweiterungen](https://datenstrom.se/de/yellow/extensions/) oder [die experimentellen Erweiterungen auf GitHub](https://github.com/topics/datenstrom-yellow) an. Es gibt mehrere Kommentar-Erweiterungen, sowohl selbst gehostet als auch mit externen Diensten. Dort sollte meines Wissens auch eine Analytics-Erweiterung mit Unterstützung mehrerer Dienste verfügbar sein. 

Ich hoffe diese Infos helfen dir für den Einstieg etwas weiter. Für konkrete Fragen erstelle wie schon erwähnt am besten eine neue Diskussion oder verwende zunächst die Suchfunktion, vieles lässt sich darüber schon beantworten. 🙂

Steffen

**Comment by l4ndfr05ch** [comment]
Hi Steffen,

vielen Dank für deine ausführliche Antwort. Ich nehme deine Hinweise gerne an, muss aber auch sagen, das ich im Vorfeld schon die Hinweis-Seiten gelesen hatte, nur ist der Durchblick am Anfang nicht ganz so einfach.

Und du hast natürlich recht, das es mit Kenntnissen zu html und css besser ist das Layout anzupassen. Ich bin auf einem guten Weg, auch wenn ich erst am Anfang stehe.

Viele Grüße,
Marko

**Comment by schulle4u** [comment]
Ja, durch die Verteilung auf GitHub und Webseite kann es manchmal für Neulinge etwas verwirrend sein. 🙂

P.S.: Wir lassen Diskussionen grundsätzlich geöffnet, damit sie für andere als beantwortet sichtbar bleiben.
